docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -115,18 +115,23 @@ Allowable values for these tags are as follows, with multiple values separated b
 | none | None of the above are used. |
 | all | All of the above (except `none`) |
 
-In the following example, everything except the content files from the package would be consumed by the project and everything except content files and analyzers would flow to the parent project.
-
 ```xml
 <ItemGroup>
     <!-- ... -->
-
+    <!-- Everything except the content files will be consumed by the project -->
+    <!-- Everythign except content files and analyzers will flow to the parent project-->
     <PackageReference Include="Contoso.Utility.UsefulStuff" Version="3.6.0">
-        <IncludeAssets>all</IncludeAssets>
+        <IncludeAssets>all</IncludeAssets> <!-- Default is `all`, can be omitted-->
```

Comment:
Should we move comment above for consistency?

Reply:
The way I saw it was that the 2 comments above talk about the overall behavior of the 3 props when combined, so the effective behavior. 
The inline one is referring specifically to `IncludeAssets`.

Reply:
cool, merge!